src/dmd/mars.d

```diff
@@ -1338,7 +1338,7 @@ extern(C) void printGlobalConfigs(FILE* stream)
                 buf.printf("%s ", flag);
         }
 
-        auto res = buf.peekSlice() ? buf.peekSlice()[0 .. $ - 1] : "(none)";
+        auto res = buf[] ? buf[][0 .. $ - 1] : "(none)";
```

Comment:
Nits: `buf[]` is not the best way to check for length 0, It returns true after writing to the buffer and resetting it with `reset()`.
`buf[][0 .. $ - 1]` could be written as `buf[0 .. $ - 1]` if `opDollar` is also defined in OutBuffer.